### PR TITLE
[12.0][MIG] mgmtsystem_audit

### PR DESCRIPTION
**Pull request by ernestotejeda**
Depends on:

- [x] https://github.com/OCA/management-system/pull/247

Cc @Tecnativa

### CODE REVIEW

**Comment by ernestotejeda** [comment]
Rebased

**Comment by pedrobaeza** [comment]
Please fix runbot warnings:
```
2019-07-25 20:19:33,443 160 WARNING openerp_test odoo.models: ir.actions.server.create() with unknown fields: body_html, condition, email_to, subject
2019-07-25 20:19:33,467 160 WARNING openerp_test odoo.models: ir.actions.server.create() with unknown fields: body_html, condition, email_to, subject
2019-07-25 20:19:33,680 160 WARNING openerp_test odoo.addons.base.models.ir_ui_view: The group base.group_extended defined in view mgmtsystem.nonconformity.form does not exist!
```

**Comment by ernestotejeda** [comment]
Changes done

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Made some squashing

File: mgmtsystem_audit/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Users must be added to the appropriate groups within openERP as follows:
```
Comment (max3903) [inline mgmtsystem_audit/readme/CONFIGURE.rst]: `within Odoo`

**Review by max3903 — APPROVED** [review]

File: mgmtsystem_audit/readme/CONFIGURE.rst (hunk 1 above)
Comment (pedrobaeza) [reply mgmtsystem_audit/readme/CONFIGURE.rst]: Changed

**Pull request merged**